docs/go-vanilla/topics/client.md

```diff
@@ -0,0 +1,24 @@
+In your Go code, we specify a TLS stack configuration for your client(s) making requests. The configuration includes 1.) root certificates of all trusted CAs for verification of the server's certificate in a pool we create. And 2.) the client's own certificate and private key for server-side client certificate verification.
+```go
+// ...
+
+caCert, _ := ioutil.ReadFile("root.crt")
```

Comment:
These actually end up getting parsed as templates (mustache) so we can sub in certificate names and filenames in our docs. So, `"root.crt"` should be subbed for `"{{ ca_cert }}"`.